### PR TITLE
Update backtrace_silencers.rb

### PR DESCRIPTION
**Pull request by andyduong1920**
## What happened
Solved #147

 

## Proof Of Work
image

### CODE REVIEW

File: config/initializers/backtrace_silencers.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+gsub_file 'config/initializers/backtrace_silencers.rb', '# Rails.backtrace_cleaner.remove_silencers!', "Rails.backtrace_cleaner.remove_silencers!\n"
+
+insert_into_file 'config/initializers/backtrace_silencers.rb', after: "Rails.backtrace_cleaner.remove_silencers!\n" do
+  <<-EOT
+  
+# Add backtrace silencer
+# Default silencer is removing backtrace involving engines files
+# Our silencer also checks for engines directory
+# Ref: https://github.com/rails/rails/blob/master/railties/lib/rails/backtrace_cleaner.rb
```
Comment (noor-ahmed) [inline config/initializers/backtrace_silencers.rb:9]: can we add `"Rails.backtrace_cleaner.remove_silencers!\n"` after it? and also use `after: "# Rails.backtrace_cleaner.remove_silencers!\n"` at line-3? just from curiosity , it is possible, right?

Comment (andyduong1920) [reply config/initializers/backtrace_silencers.rb:9]: Sorry @noor-ahmed I'm not sure I understand your question clearly, do you mean, can we merge into 1 command 🤔 , something like this?

```
gsub_file 'config/initializers/backtrace_silencers.rb', '# Rails.backtrace_cleaner.remove_silencers!', """
  Rails.backtrace_cleaner.remove_silencers!
  
  # Add backtrace silencer
  # Default silencer is removing backtrace involving engines files
  # Our silencer also checks for engines directory
  # Ref: 
 
 https://github.com/rails/rails/blob/master/railties/lib/rails/backtrace_cleaner.rb
  ....
"""
```

Comment (noor-ahmed) [reply config/initializers/backtrace_silencers.rb:9]: @andyduong1920 Exactly!! can we do it?

Comment (noor-ahmed) [reply config/initializers/backtrace_silencers.rb:9]: @andyduong1920 Yeah something like this,

after line 4 we add `Rails.backtrace_cleaner.remove_silencers!`, is it okey? I'm not sure though 😕

Comment (andyduong1920) [reply config/initializers/backtrace_silencers.rb:9]: Yes, it works @noor-ahmed

Comment (andyduong1920) [reply config/initializers/backtrace_silencers.rb:9]: Fixed in https://github.com/nimblehq/rails-templates/pull/233/commits/142d2b7226813e9fe37aec90e0fc52c3da5a7275 @noor-ahmed

**Review by noor-ahmed — APPROVED** [review]

**Pull request merged**